charts/argocd-apps/ci/applicationsets-values.yaml

```diff
@@ -1,8 +1,7 @@
 # Test with applicationsets
 
 applicationsets:
-- name: guestbook
-  namespace: default # Only for test purpose.
+- name: applicatioset
```

Comment:
Small typo here - why this was changed from Argo sample app?

Reply:
Thank you. Fixed it!
Just to avoid a name collision.